gazebo_plugins/include/gazebo_plugins/gazebo_ros_hand_of_god.hpp

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019 Open Source Robotics Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GAZEBO_PLUGINS__GAZEBO_ROS_HAND_OF_GOD_HPP_
+#define GAZEBO_PLUGINS__GAZEBO_ROS_HAND_OF_GOD_HPP_
+
+#include <gazebo/common/Plugin.hh>
+
+#include <memory>
+
+namespace gazebo_plugins
+{
+class GazeboRosHandOfGodPrivate;
+
+/// Drives a floating object around based on the location of a TF frame.
+/// It is useful for connecting human input
```

Comment:
Since we're touching the plugin, mind adding some documentation which was lacking in ROS 1? Explaining things like

* The plugin expects a TF `frame_id + "_desired"`
* The plugin broadcasts TF `frame_id + "_actual"`